README.rst

```diff
@@ -2,6 +2,46 @@
 ts_aos_utils
 ############
 
+Requirements
+------------
+
+This package requires some extra repositories to be cloned locally so it can grab some constants and some look-up tables. Here is a list of which repos are required to run this notebook:
```

Comment:
Please help to follow the Markdown syntax that each new sentence would be a new line. You can see that this will not change the format.